moveit_planners/stomp/industrial_collision_detection/include/industrial_collision_detection/collision_detector_allocator_industrial.h

```diff
@@ -0,0 +1,43 @@
+/**
+ * @file collision_detector_allocator.ch
+ * @brief An allocator for Industrial Moveit FCL collision detectors
```

Comment:
This loads the CollisionRobotIndustrial and CollisionWorldIndustrial classes instead of the FCL classes.